src/coprocessor/codec/mysql/duration.rs

```diff
@@ -655,26 +691,26 @@ mod tests {
             (b"232 10", 0, None),
             (b"-232 10", 0, None),
             (b"00:00:00.1", 0, Some("00:00:00")),
-            (b"00:00:00.1", 1, Some("00:00:00.1")),
-            (b"00:00:00.777777", 2, Some("00:00:00.78")),
+            (b"00:00:00.1", 1, Some("00:00:00.100000")),
```

Comment:
I believe this change is incorrect. for fsp = 1, we should return at most one fractional digit.

Reply:
In `MySQL`, whatever the value of `fsp` is, it's printed in the following way:
```
mysql iosmanthus@(none):test> create table duration (time Time(1))                                                                                    
Query OK, 0 rows affected
Time: 0.025s
mysql iosmanthus@(none):test> insert into duration values ("0.999")                                                                                   
Query OK, 1 row affected
Time: 0.012s
mysql iosmanthus@(none):test> select * from duration                                                                                                  
+----------------+
| time           |
+----------------+
| 0:00:00.900000 |
+----------------+
1 row in set
Time: 0.026s
```